### ISSUE TITLE
Master branch doesn't build AL plugin on Windows 10 

### ISSUE THREAD

**Issue by daniele-niero** [Autodesk/maya-usd#56]
Animal Logic's plugin doesn't really like to be built on Windows:
https://github.com/AnimalLogic/AL_USDMaya/issues/76

I'm not sure if there is a solution for this in a more recent version of USD. Version 19.05, the one supposed to be used with the master branch of maya-usd, certainly have this problem.

https://github.com/PixarAnimationStudios/USD/pull/840/commits/bc78ee9ed54768259258fa3eb76049da63533f78
It looks like there was a push request specifically for allow AL-USD to build, however it din't pass.

Also I don't get this Autodesk plug-in, right now it just build the Pixar's and the Animal-Logic's... as far as I can tell, I don't see any attempt to merge them in a unique "best-of-the-two-world" plug-in...

**Comment by BigRoy** [comment]
> Also I don't get this Autodesk plug-in, right now it just build the Pixar's and the Animal-Logic's... as far as I can tell, I don't see any attempt to merge them in a unique "best-of-the-two-world" plug-in.

This work is being done in `refactoring_sandbox` branch where the code is based on a uniform parent code base where the variety of plug-ins inherit from. In the end it mean the basic functionality would be shared by all. But yes, the `master` branch currently has none of that as the work is still being done before there's a stable basis.

The approach/roadmap is somewhat described in [this `maya-usd` talk.](https://vimeo.com/365638295)

> I'm not sure if there is a solution for this in a more recent version of USD. Version 19.05, the one supposed to be used with the master branch of maya-usd, certainly have this problem.

I was able to build the master branch on Windows with the AL plug-in. So it should be doable. However, I did follow some notes of 'fixes' that might be related to it not working on your end. More info here: https://gist.github.com/BigRoy/4593ef47f39a3589308454088a0a2753#file-maya-usd-compile-on-windows-10-notes-txt

Hopefully that gets you there to compiling things on Windows. Note that some fixes on the USD side and the `refactoring_sandbox` branch seem to be focused on fixing these things so wouldn't need these "hacks". But those are not in `master` yet and I'm also not sure they currently will fix all of these things. :)
